### PR TITLE
Portuguese translations

### PR DESCRIPTION
**Pull request by zecarlos1957**
Portuguese Translation.

## Proposed changes
syssetup  update portuguese translation.

### CODE REVIEW

File: dll/win32/syssetup/syssetup.rc [hunk 1]
```diff
@@ -108,6 +108,9 @@ IDR_GPL RT_TEXT "COPYING"
 #ifdef LANGUAGE_PT_BR
     #include "lang/pt-BR.rc"
 #endif
+#ifdef LANGUAGE_PT_PT
+    #include "lang/pt-PT.rc
```
Comment (Fraizeraust) [inline dll/win32/syssetup/syssetup.rc]:
```suggestion
    #include "lang/pt-PT.rc"
```
The last double quotes are missing which is the cause why TravisCI fails to compile the build.

**Comment by Fraizeraust** [comment]
I'd recommend that you add descriptions of your commits in a concise way. Like:
```
[COMPONENT_NAME] XYZ translation update
```

Could be also `Add XYZ translation` depending on what is your work about. `COMPONENT_NAME` is what you've translated (for example **USETUP** which is the 1st stage setup module or **MAGNIFY**, the Magnifier application).

File: dll/win32/syssetup/syssetup.rc (hunk 1 above)
Comment (zecarlos1957) [reply dll/win32/syssetup/syssetup.rc]: Thanks for the suggestion Fraizeraust. I have already amended the error.

**Comment by HBelusca** [comment]
Committed, thanks!

**Pull request merged**